Week7Threading/Week7Threading/Program.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Diagnostics;
+
+namespace Week7Threading
+{
+    /// <summary>
+    /// This class contains the Main method and all associated code.
+    /// </summary>
+    public class Program
+    {
+        /// <summary>
+        /// This method asks the user for a number, then prints out a list of prime numbers up to the given number.
+        /// </summary>
+        /// <param name="args"></param>
+        static void Main(string[] args)
+        {
+            bool running = true;
+            Console.WriteLine("This program creates two lists of prime numbers up to a user specified number");
+            Console.WriteLine("using both parallel and sequential for loops");
+            while (running)
+            {
+                Console.WriteLine("User number:");
+                try
+                {
+                    int givenNumber = Convert.ToInt32(Console.ReadLine());
+                    Console.WriteLine("All prime numbers up to " + givenNumber + " parallel:");
+                    Stopwatch parallelStopwatch = new Stopwatch();
+                    parallelStopwatch.Start();
+                    // This statement makes use of the parallel foreach loop.
+                    foreach (int num in new PrimeCalculator().CalculatePrimesOfNumberFast(givenNumber))
```

Comment:
Since "Fast" and "Slow" are not actually easy to guarantee, it would be better to name the methods more in terms of their functionality. Maybe something like "CalculatePrimesInParallel". Even that is not great because it conveys too much about how the method is implemented, but since you need to disambiguate in this case between parallel and synchronous operations, it could work.